src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -760,6 +760,35 @@ static void Goo()
                 """, "cref", "langword", "href");
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72259")]
+        public async Task SeeAttributeNames2()
+        {
+            var text = """
+                class C
+                {
+                    /// <summary>
+                    /// <see l$$=""/>
+                    /// </summary>
+                    static void Goo()
+                    {
+                    }
+                }
+                """;
+            var expected = """
+                class C
+                {
+                    /// <summary>
+                    /// <see langword=""/>
+                    /// </summary>
+                    static void Goo()
+                    {
+                    }
+                }
+                """;
+
+            await VerifyProviderCommitAsync(text, "langword", expected, null);
+        }
```

Comment:
can you add a similar test in CompletionCommandHandlerTests?  Also, please test the case where the user has typed *nothing* but brings up completion and completes out one of these words.  like `<see $$=""/>`.  Note: all these tests i'm asking for should go into CompletionCommandHandlerTests, not the provider tests.  thanks.

Reply:
@CyrusNajmabadi I moved the tests to CompletionCommandHandlerTests, but adding the test to type nothing seems like it won't help much.

If you type nothing and invoke the completion, it doesn't actually suggests you attributes name, it would rather suggests you tags (so it would actually suggest me a `see` tag here even though I just wrote one). This could be a separate issue, but for testing this behavior in particular, it didn't seem like it would have helped.

If a test should still be added, what are your expectations of it?

Reply:
Done.